lib/cookie-service/cookie.service.ts

```diff
@@ -124,6 +126,10 @@ export class CookieService {
       cookieString += 'secure;';
     }
 
+    if (sameSite) {
```

Comment:
I know the style guide is a little odd 😄 but this should be:

```
  if ( sameSite ) {
```

Reply:
Done :)